.github/workflows/pr_agent.yml

```
@@ -0,0 +1,24 @@
+    if: ${{ github.event.sender.type != 'Bot' }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      issues: write
+    steps:
+      - name: PR Agent action step
+        id: pragent
+        uses: qodo-ai/pr-agent@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)